config/account.go

```diff
@@ -81,6 +93,30 @@ func (pf *AccountPriceFloors) validate(errs []error) []error {
 		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.max_schema_dims should be between 0 and 20`))
 	}
 
+	if pf.Fetch.Period > pf.Fetch.MaxAge {
+		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.fetch.period_sec should be less than account_defaults.price_floors.fetch.max_age_sec`))
+	}
+
+	if pf.Fetch.Period < 300 {
+		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.fetch.period_sec should not be less than 300 seconds`))
+	}
+
+	if !(pf.Fetch.MaxAge >= 600 && pf.Fetch.MaxAge < math.MaxInt32) {
+		errs = append(errs, fmt.Errorf(`account_defaults.price_floors.fetch.max_age_sec should not be less than 600 seconds and greater than maximum integer value`))
+	}
```

Comment:
We are we opinionated about enforcing a minimum value?

Reply:
Value validations
enforce-floors-rate >=0 and <= 100
fetch.period-sec must be <= fetch.max-age-sec
fetch.period-sec >= 300 (cannot poll more often than every 5 min)
fetch.max-age-sec >= 600 (must allow data to be valid for at least 10 min)
fetch.max-age-sec < MAXINT
fetch.timeout-ms must be > 10 and < 10,000
fetch.max-rules >= 0 and < MAXINT
fetch.max-file-size-kb >= 0 and < MAXINT
fetch.max-schema-dims >= 0 and < 20
max-rules >= 0 and < MAXINT
max-schema-dims >= 0 and < 20

Reply:
From The PRD